src/components/profile/widgets/widget-photo.tsx

```diff
@@ -10,12 +10,46 @@ interface PhotoWidgetType {
 }
 
 export const PhotoWidget: React.FC<PhotoWidgetType> = ({ content }) => {
+  // stylings for cropped images derived from official react-easy-crop examples
+  // reference: https://codesandbox.io/p/sandbox/react-easy-crop-with-live-output-kkqj0?file=%2Fsrc%2FApp.jsx%3A49%2C11
+  // this is the only approach that seems to handle images of any dimensions.
+
+  const croppedArea =
+    (content as PhotoWidgetContentType).croppedArea ?? undefined;
+  const isCropped = (content as PhotoWidgetContentType).isCropped;
+
```

Comment:
```suggestion
  const croppedArea = content.croppedArea;
  const isCropped = content.isCropped;
```

Doesn't look like this cast is needed